lib/sshkit/utils.ex

```diff
@@ -2,4 +2,11 @@ defmodule SSHKit.Utils do
   def shellescape(value), do: value
 
   def shellquote(value), do: value
+
+  def strings_to_charlists(opts) when is_list(opts) do
+    Enum.map(opts, fn {k, v} -> {k, charlist(v)} end)    
```

Comment:
I wonder whether we should handle cases, where binaries are present in more deeply nested structures. For instance, Erlang's `:ssh.connect` accepts these options:

- `{:pref_public_key_algs, list()}` where `list()` is expected to be a list of `charlists`
- `{:preferred_algorithms, algs_list()}`
- …

(I think those are the only ones but I'm not 100% sure?)

Is there an elegant way of doing a "deep" conversion? Does it make sense to handle such cases?